packages/editor/src/components/commands/index.js

```diff
@@ -253,7 +253,18 @@ function useEditorCommandLoader() {
 			},
 		} );
 	}
-
+	const isInSiteEditor =
+		window.location.pathname.includes( 'site-editor.php' );
```

Comment:
```suggestion
	const isSiteEditor = getPath( window.location.href )?.includes(
		'site-editor.php'
	);
```

Please import `getPath` from the `@wordpress/url` package.